pkg/agent/plugin/workloadattestor/windows/windows_windows.go

```diff
@@ -60,12 +82,28 @@ func (p *Plugin) Attest(ctx context.Context, req *workloadattestorv1.AttestReque
 		selectorValues = addSelectorValueIfNotEmpty(selectorValues, "group_name", group)
 	}
 
+	// obtaining the workload process path and digest are behind a config flag
```

Comment:
I think that this is true for the unix attestor but I'm not sure that there is an extra permissions requirement in Windows (i.e. the QueryFullProcessImageName function needs that the handle must be created with the PROCESS_QUERY_INFORMATION or PROCESS_QUERY_LIMITED_INFORMATION access right, which we have).

I still think that this should be behind a config flag due to the cost of calculating the SHA256 digest, that may not be needed by everyone. But looks like this comment needs to be updated.